client/app/js/directives.js

```diff
@@ -59,8 +63,18 @@ GL
           }
           $scope.isDatePickerOpened = false;
         }
+
+         if ($scope.daterangePickerModel.start && $scope.daterangePickerModel.end) {
+           $scope.obj.value1 = true;
+         }else{
+           $scope.obj.value1 = false;
+         }
+
          $scope.onDatechange($scope.dataRangeFilter);
       });
+      $scope.checkFilter = function() {
+        return $scope.obj.value1 === true;
```

Comment:
@dev-genesistechnologies : i will simply remove this variable and change this function to do:

return "return $scope.daterangePickerModel.start && $scope.daterangePickerModel.end;"